packages/node-cli/src/commands/ImportObjectsCommandGenerator.js

```diff
@@ -125,18 +124,18 @@ module.exports = class ImportObjectsCommandGenerator extends BaseCommandGenerato
 					{ name: NodeTranslationService.getMessage(YES), value: true },
 					{ name: NodeTranslationService.getMessage(NO), value: false },
 				],
-				validate: fieldValue => showValidationResults(fieldValue, validateArrayIsNotEmpty),
+				validate: (fieldValue) => showValidationResults(fieldValue, validateArrayIsNotEmpty),
```

Comment:
There is no need for the parenthesis when an arrow function is using only one argument. There is a lot of cases like this in the review. Was it done by your prettier? I personally don't like it, but I'll stick to what ever we decide as a team.

Reply:
Yeah, it was done by prettier.. Let's discuss in Slack for going faster